l10n_br_account/l10n_br_account.py

```diff
@@ -450,6 +451,40 @@ class L10n_brTaxDefinition(models.Model):
         related='tax_id.company_id', readonly=True,
         store=True, string='Empresa')
 
+
+class AccountInvoiceTax(models.Model):
+
+    _inherit = "account.invoice.tax"
+
+    deduction_account_id = fields.Many2one('account.account',
+                                           'Tax Account for Deduction')
+
+    @api.model
+    # create counterpart credit tax
+    def move_line_get(self, invoice_id):
+        res = super(AccountInvoiceTax, self).move_line_get(invoice_id)
+        inv = self.env['account.invoice'].browse(invoice_id)
+        pudb.set_trace()
+        self._cr.execute(
+            'SELECT * FROM account_invoice_tax WHERE invoice_id = %s',
+            (invoice_id,)
+        )
+        for row in self._cr.dictfetchall():
+            if row['amount'] and row['tax_code_id'] and\
+                    row['tax_amount'] and row['deduction_account_id']:
+                res.append({
+                    'type': 'tax',
+                    'name': row['name'],
+                    'price_unit': row['amount'],
+                    'quantity': 1,
+                    'price': -row['amount'] or 0.0,
+                    'account_id': row['deduction_account_id'],
+                    'tax_code_id': False,
+                    'tax_amount': False,
+                    'account_analytic_id': row['account_analytic_id'],
+                })
+        return res
```

Comment:
Melhorar essa parte do código, levar em consideração o comentário:

"Why load the invoice and after do a 'select * ' in the same table, I think it's better in this case not bypass the ORM"